android/src/main/java/com/example/flutter_braintree/FlutterBraintreeDropIn.java

```diff
@@ -141,6 +141,7 @@ private static void readPayPalParameters(DropInRequest dropInRequest, MethodCall
 
   @Override
   public boolean onActivityResult(int requestCode, int resultCode, Intent data)  {
+    if(activeResult == null){return true;}
```

Comment:
Could you please remove the `{}` and add some spaces so it says:
```java
 if (activeResult == null) return true;
```
That would be great, thank you.